core/src/main/java/org/apache/accumulo/core/gc/ReferenceDirectory.java

```diff
@@ -21,13 +21,22 @@
 import org.apache.accumulo.core.data.TableId;
 
 /**
- * Part of the Tablet File path that is definitely a directory.
+ * A GC reference to a Tablet directory, like t-0003.
  */
-public class ReferenceDirectory extends Reference {
-  public final String tabletDir; // t-0003
+public class ReferenceDirectory extends ReferenceFile {
+  private final String tabletDir; // t-0003
 
   public ReferenceDirectory(TableId tableId, String dirName) {
     super(tableId, dirName);
```

Comment:
In this case, the `metadataEntry` is only the dirname, like `t-0003` ? Or, is the example wrong?

Reply:
The metadataEntry will look like: `1;2cccccccccccccd1 srv:dir []   t-000003z`. So technically, the dirName is only part of the metadaEntry.

Reply:
Ok, thanks for the clarification.